### ISSUE TITLE
Slight difference in code vs paper

### ISSUE THREAD

**Issue by sairaamVenkatraman** [ssrp/Multi-level-DCNet#2]
In your ACCV paper, you had mentioned that you used no augmentation for fashionmnist. However, your code and comments on the fashionmnist site tell that you have used random translations by up to 2 pixels and random horizontal flips. Please clarify.

**Comment by ssrp** [comment]
Hi @sairaamVenkatraman, many thanks for your question. In the code, we have added the functionality of the data augmentation but we are not using it for the experiments in the paper. Please note that the shift_fraction could be changed to anything you want and run the code (set to 0 for no augmentation). I have updated the default value in the code, thanks!

**Comment by sairaamVenkatraman** [comment]
Thanks for the prompt reply. My question was actually based on your submission to the fashionmnist github page. Specifically, this https://github.com/zalandoresearch/fashion-mnist/issues/140

In your paper, however, you have mentioned that no augmentation was used. Hence, the question.

**Comment by ssrp** [comment]
Thanks again for your reply. That's a mistake on my end, let me confirm the exact parameters and I'll get back to you soon. I am very sure that we used the same parameters (and augmentation techniques) for both baseline CapsNets and DCNets, but I'll confirm if we used data augmentation soon.
Sorry for the inconvenience. Re-opening this issue.

**Comment by sairaamVenkatraman** [comment]
Thank you!

**Comment by ssrp** [comment]
Hi @sairaamVenkatraman, thank you for your message. We have realised that there was a mistake in the ACCV paper. We used the exact augmentation that was used in CapsNet baseline model (as mentioned in the NIPS 2017 paper) to make the results comparable. So all the models are trained with the same augmentation (also for FashionMNIST).
We'll mention this as an erratum on the repository and update the paper linked to the repo.
Thanks for bringing up this question!
